data/events/writethedocs-eu-2018.json

```diff
@@ -0,0 +1,20 @@
+{
+    "accessibility": "",
+    "code_of_conduct": "http://www.writethedocs.org/code-of-conduct/",
+    "event_end": "2018-09-13",
+    "event_start": "2018-09-11",
```

Comment:
According to http://www.writethedocs.org/conf/eu/2017/news/recap-retrospective/
the dates are:
Prague conference, 9-11 September 2018

Reply:
Fantastic catch - I thnk the reversal of the entries caused by brain to misfire.  As this PR merged, I'll send a fix up.